modules/swift/examples/code_snippets/SampleCodeTest.swift

```diff
@@ -1209,2824 +1208,1603 @@ class SampleCodeTest {
+            return nil
+        }
+        // Store all Certs for Id in Keychain:
+        var i = 0;
+        for cert in certChain {
+            let params: [String : Any] = [
+                String(kSecClass):      kSecClassCertificate,
+                String(kSecValueRef):   cert,
+                String(kSecAttrLabel):  "doco-sync-server"
+            ]
+            osStatus = SecItemAdd(params as CFDictionary, nil)
+            if osStatus != errSecSuccess {
+                print("Unable to store certs")
                 return nil
             }
-       // Store all Certs for Id in Keychain:
-       var i = 0;
-       for cert in thisCertChain! {
-            let params: [String : Any] = [
-                String(kSecClass):      kSecClassCertificate,
-                String(kSecValueRef):   cert,
-                String(kSecAttrLabel):  "doco-sync-server"
-                ]
-            kcStatus = SecItemAdd(params as CFDictionary, nil)
-                if kcStatus != errSecSuccess {
-                    print("Unable to store certs")
-                    return nil
-                }
-            i=i+1
+            i=i+1
+        }
+#else
+        let query: [String : Any] = [
+            String(kSecClass):          kSecClassCertificate,
+            String(kSecValueRef):       certs[0]
+        ]
+        
+        let update: [String: Any] = [
+            String(kSecClass):          kSecClassCertificate,
+            String(kSecValueRef):       certs[0],
+            String(kSecAttrLabel):      label
+        ]
+        
+        osStatus = SecItemUpdate(query as CFDictionary, update as CFDictionary)
+        if osStatus != errSecSuccess {
+            print("Unable to update certs \(osStatus)")
+            return nil
+        }
+#endif
+        // end::p2p-tlsid-store-in-keychain[]
+        
+        // tag::p2p-tlsid-return-id-from-keychain[]
+        
+        // RETURN A TLSIDENTITY FROM THE KEYCHAIN FOR USE IN CONFIGURING TLS COMMUNICATION
+        return try TLSIdentity.identity(withIdentity: secIdentity, certs: [certChain[1]])
+        // end::p2p-tlsid-return-id-from-keychain[]
+    }
+    
+    func dontTestDeleteIDFromKeychain() throws {
+        // tag::p2p-tlsid-delete-id-from-keychain[]
+        
+        try TLSIdentity.deleteIdentity(withLabel: "doco-sync-server")
+        
+        // end::p2p-tlsid-delete-id-from-keychain[]
+        
+    }
+    
+    func dontTestReplicatorConfigSelfCert() throws {
+        let target = DatabaseEndpoint(database: otherDB)
+        var config = ReplicatorConfiguration(database: database, target: target)
+        // tag::old-p2p-act-rep-config-self-cert[]
+        // acceptOnlySelfSignedServerCertificate = true -- accept Slf-Signed Certs
+        //        FIXME: !!
+        //          `disableTLS` is for listener and it is 'false' by default.
+        //          `acceptOnlySelfSignedServerCertificate` is for replicator
+        //        config.disableTLS = false
```

Comment:
This whole tagged block can be removed @jayahariv 

        // tag::old-p2p-act-rep-config-self-cert[]

        // end::old-p2p-act-rep-config-self-cert[]